packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -87,6 +89,29 @@ export async function activate(extensionContext: vscode.ExtensionContext) {
     languageClient = await languageServer.createLanguageServer(
       extensionContext
     );
+
+    // TODO: unsure when/how this 'error' handler is triggered
```

Comment:
@lukeis the base class of EventEmitter handles running the supplied arrow functions each time the error handler calls a `this.emit(...)` function.